### PR TITLE
Update crossterm_test.yml

### PR DESCRIPTION
**Pull request by TimonPost**
#205

### CODE REVIEW

File: .github/workflows/crossterm_test.yml [hunk 1]
```diff
@@ -17,9 +20,20 @@ jobs:
       with:
         rust-version: ${{ matrix.rust }}
     - uses: actions/checkout@master
+    - name: Add Component
```
Comment (zrzka) [inline .github/workflows/crossterm_test.yml]: See [this](https://github.com/hecrj/setup-rust-action/blob/master/action.yml#L8-L10). You can use `with.components: 'rustfmt'`.

File: .github/workflows/crossterm_test.yml [hunk 2]
```diff
@@ -1,25 +1,52 @@
...
+    - name: Debug Information
+      run: | 
+        cargo fmt --version
+        rustup --version
+        rustc --version
+      
     - name: Run Build
       run: cargo build
+      
     - name: Check Examples
```
Comment (zrzka) [inline .github/workflows/crossterm_test.yml]: This step is redundant, `cargo test --all` covers this.

**Comment by zrzka** [comment]
What are you trying to achieve? Check [this workflow](https://github.com/zrzka/crossterm/blob/zrzka/gh-actions/.github/workflows/crossterm_test.yml). It:

* uses stable & nightly
* uses rustfmt on both
* checks formatting on both
* run tests on both

And what's more important, it requires stable to pass and nightly is allowed to fail.

**Comment by TimonPost** [comment]
Thanks, I will try that.

**Comment by zrzka** [comment]
Seems that GH actions are kind of broken. I've got lot of canceled builds without a good reason. No idea what's going on, still beta, ...

**Comment by TimonPost** [comment]
It broken on macos fromat check

**Comment by zrzka** [comment]
Hmm ...

```bash
(zrzka/fix-cargo-fmt ✓) crossterm /bin/bash -e cargo fmt --all -- --check
/Users/robertvojta/.cargo/bin/cargo: /Users/robertvojta/.cargo/bin/cargo: cannot execute binary file
(zrzka/fix-cargo-fmt ✓) crossterm cargo fmt --all -- --check
(zrzka/fix-cargo-fmt ✓) crossterm echo $?
0
(zrzka/fix-cargo-fmt ✓) crossterm 
```

**Comment by zrzka** [comment]
It's weird, because when you check _Check code formatting_ [here](https://github.com/zrzka/crossterm/runs/222137493), it was started, but then canceled.

**Comment by TimonPost** [comment]
From what I see here, is that it stops after a few errors when checking for fmt.
 

https://github.com/TimonPost/crossterm/pull/215/checks?check_run_id=222174585#step:5:4

**Comment by zrzka** [comment]
Yeah, if I understand it correctly, the build was canceled by GH Actions, not that rustfmt fails or anything like this.

**Comment by zrzka** [comment]
We have to sign up again for GH actions to continue with testing. Because we moved the repo to the organization, the whole `crossterm-rs` org should ask for it.

**Comment by TimonPost** [comment]
Done, I think that once everything is merged we should try this PR again.

**Comment by zrzka** [comment]
Closing this PR. Changes:

* reenabled GitHub actions
* made the job name shorter to fit the column width
* added packaging test (stable only)
* synced steps with `.travis.yml`

Seems that it works properly. Check [this test suite](https://github.com/crossterm-rs/crossterm/commit/5904a2a10677a36084cbbc08e9589ea6b06b47cf/checks?check_suite_id=271677333).

GitHub Actions are way faster than Travis CI, but still beta. So, keeping the `.travis.yml` there, which should be still treated as a mandatory test for a PR.

**Pull request closed without merging**